File: src/S1/BOOfNP2/TS/main.ts
function showNames(students: string[]): void {
    for (let i = 0; i < students.length; i++) {
        console.log(students[i].toUpperCase());
    }

    for (let i = 0; i < students.length; i++) {
        console.log(students[i].toLowerCase());
    }
}

const studentNames1: string[] = [
    "Alice", "Bob", "Charlie", "David", "Eva", "Frank", "Grace", "Hannah",
    "Isaac", "Julia", "Kevin", "Lily", "Mike", "Nina", "Oliver", "Pamela",
    "Quincy", "Rachel"
];

// Example usage:
showNames(studentNames1);

File: src/S1/BOOfNP2/Python/main.py
def show_names(students):
    for student in students:
        print(student.upper())

    for student in students:
        print(student.lower())

# Example usage:
student_names = [
    "Alice", "Bob", "Charlie", "David", "Eva", "Frank", "Grace", "Hannah",
    "Isaac", "Julia", "Kevin", "Lily", "Mike", "Nina", "Oliver", "Pamela",
    "Quincy", "Rachel"
]

show_names(student_names)
